envrc.el

```diff
@@ -126,6 +126,14 @@ To access `envrc-command-map' from this map, give it a prefix keybinding,
 e.g. (define-key envrc-mode-map (kbd \"C-c e\") \\='envrc-command-map)"
   :type '(restricted-sexp :match-alternatives (keymapp)))
 
+(defcustom envrc-remote nil
+  "Whether or not to enable direnv over TRAMP."
+  :type 'boolean)
+
+(defcustom envrc-supported-tramp-methods '("ssh" "sshx")
```

Comment:
I propose adding `"scp" "scpx"` here, since they differ only in file transfer method, everything else is the same as for `ssh[x]`. At least for me, it works well for scpx.